spacy/tests/serialize/test_serialize_kb.py

```diff
@@ -86,12 +86,19 @@ def test_serialize_subclassed_kb():
     [nlp]
     lang = "en"
     pipeline = ["entity_linker"]
+    
+    [default_values]
+    custom_field = 666
 
     [components]
 
     [components.entity_linker]
     factory = "entity_linker"
-
+    
+    [components.entity_linker.generate_empty_kb]
+    @misc = "spacy.CustomEmptyKB.v1"
+    custom_field = ${default_values.custom_field}
```

Comment:
As a nitpick, I would actually have the `spacy.CustomEmptyKB.v1` (note that this shouldn't be prefixed with spaCy as it mimicks a user case) not take the `custom_field` parameter and just set it so some random int like 0. Then the test double checks that the correct `custom_field` had actually been loaded, not the default one from the empty factory.

Reply:
That was a good catch, we kinda cheated so far on this test - we didn't (de-)serialize `custom_field`. I added custom `.to_disk()` and `.from_disk()` implementations that piggyback on the `InMemoryLookupKB` implementations. It's not super elegant, but I couldn't think of a better way without diving too deep into the `InMemoryLookupKB` serialization logic. If you have a better idea, I'd be happy to hear it.

Done in https://github.com/explosion/spaCy/pull/12320/commits/322b0050c40fcc40b18fccd651442d72d070766b.